docs/01-app/02-guides/backend-for-frontend.mdx

```diff
@@ -67,7 +67,7 @@ export function POST(request: Request) {
 ```js filename="/app/api/route.js" switcher
 import { submit } from '@/lib/submit'
 
-export function POST(request) {
+export async function POST(request) {
```

Comment:
This fixes the JavaScript file only, please also fix the TypeScript one in line 54 ^

Reply:
Thanks! It should be fixed now :)